### PR TITLE
Adding XRegistry art

### PR DESCRIPTION
**Pull request by nate-double-u**

### CODE REVIEW

File: projects/xregistry/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+
+## XRegistry Artwork
+
+This directory contains artwork in standard formats for XRegistry. We've prepared artwork in 2 formats (PNG & SVG), 3 layouts:
+
+- horizontal (also known as landscape format),
+- stacked (which is closer to square), and
```
Comment (Cmierly) [inline projects/xregistry/README.md:7]: "Which has the simplified logo over of the project name"

might be a better description of the stacked format

Comment (nate-double-u) [reply projects/xregistry/README.md:7]: Good suggested, updated. Thanks!

File: projects/xregistry/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ This directory contains artwork in standard formats for XRegistry. We've prepare
 
 - horizontal (also known as landscape format),
 - stacked (which is closer to square), and
-- icon (which does not include the name and is square).
+- icon (which has the simplified logo over of the project name).
```
Comment (Cmierly) [inline projects/xregistry/README.md:8]: oops, "of" shouldn't be there

Comment (nate-double-u) [reply projects/xregistry/README.md]: I also goofed on the line to be changed. I've corrected both now 😅

**Review by Cmierly — APPROVED** [review]

**Comment by Cmierly** [comment]
lgtm

**Pull request merged**